_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -84015,14 +84015,14 @@
 	tag = ""
 	},
 /obj/structure/table,
-/obj/item/t_scanner,
-/obj/item/radio/headset/headset_eng,
-/obj/item/multitool{
-	pixel_x = 5
-	},
```

Comment:
Did you purposely remove these things to make room for airlock painter?

Reply:
Yes I did.

Reply:
should include in CL